### DISCUSSION TITLE
avoid xref so less storage is needed?

### DISCUSSION THREAD

**Discussion by Shooter3k** [oracle/opengrok#3946] [Q&A] [answered]
Is there any way to turn off the xref data/folder if cross reference is not needed within the code? I'm wondering because of the duplicate storage that is needed for the /xref/ folder

**Comment by vladak** [comment, marked as answer]
Yep, it's called "economy mode" and can be set by using `-e` (`--economical`) indexer option or `generateHtml` using indexer configuration (https://github.com/oracle/opengrok/wiki/Indexer-configuration).
